projects/observability/src/pages/explorer/explorer-dashboard-builder.ts

```diff
@@ -239,6 +239,19 @@ export class ExplorerDashboardBuilder {
               type: 'api-trace-navigation-handler'
             }
           },
+          {
+            type: 'table-widget-column',
+            title: 'Errors',
+            width: '180px',
+            filterable: true,
+            value: {
+              type: 'attribute-specification',
+              attribute: 'errorCount'
```

Comment:
I think the name of an attribute is `apiTraceErrorSpanCount` in API_TRACE scop
cc: @JBAhire

Reply:
While using the attribute named `errorCount`, I am getting the data but not getting any data while using `apiTraceErrorSpanCount`

Reply:
That's weird because I can see data in Pinot as well as filters on explorer and service screen. Maybe the data you are trying with doesn't have any error in trace? 

<img width="1783" alt="Screenshot 2021-04-19 at 6 29 43 PM" src="https://user-images.githubusercontent.com/26570044/115240383-540ac100-a13d-11eb-95e0-fd1289d9d712.png">
<img width="1783" alt="Screenshot 2021-04-19 at 6 29 28 PM" src="https://user-images.githubusercontent.com/26570044/115240394-55d48480-a13d-11eb-98f5-362fcc6ff3bb.png">
<img width="1425" alt="Screenshot 2021-04-19 at 6 28 57 PM" src="https://user-images.githubusercontent.com/26570044/115240399-5705b180-a13d-11eb-9b65-cb607b600894.png">

Reply:
errorCount is a different thing - it will be 1 if the api trace is overall in error, 0 otherwise (used for calculating error metrics)

Reply:
It does work with `apiTraceErrorSpanCount` now.

Reply:
referennce: https://github.com/hypertrace/hypertrace-ui/pull/786